app.js

```diff
@@ -44,23 +52,27 @@ const defaultForecast = function() {
   };
 
   location().then(function (userLocation) {
-    return forecast(userLocation.city);
+    return forecast(userLocation.city, unit);
   }).then(function (forecastWeather) {
     console.log(forecastWeather);
   }).catch(function (error) {
     console.log(error);
   });
 }
 
```

Comment:
This if statement is all I changed. Since the default is F, all we really need to do is check if it's a string and it matches ```'C'```. We don't care about matching ```'F'```. Let me know if you disagree and we can take a deeper look!